package.json

```diff
@@ -13,6 +13,8 @@
     ]
   },
   "scripts": {
+    "sort-json": "npx sort-package-json 'packages/*/package.json' 'package.json' 'examples/cactus-*/package.json' 'extensions/*/package.json' && yarn run sort-json-check",
```

Comment:
@Leeyoungone If it is possible, I'd prefer to have the package.json sorting be a part of the existing `custom-checks` npm script. That way the CI script does not need a brand new job definition (it doesn't need any changes because the `custom-checks` script is already called by the CI as of right now).
For this to happen you'd need to use the programmatic (library) API of the `sort-package-json` package instead of the CLI.

If there was some specific reason why you went with the CLI instead, then let's talk first about what that was exactly, maybe I can help resolve it or just admit defeat and keep with the CLI.